packages/dmn-js-shared/src/features/data-types/DataTypes.js

```diff
@@ -17,14 +17,16 @@ const DEFAULT_DATA_TYPES = [
  *
  * // The data types will include multiple number types: integer, long, and double.
  * const editor = new DmnJS({
- *   dataTypes: [
- *     'string',
- *     'boolean',
- *     'integer',
- *     'long',
- *     'double',
- *     'date'
- *   ]
+ *   common: {
```

Comment:
why not also have an integration test case for this (e.g., `should load customData types passed via <common.dataTypes>`)?

Reply:
OK but such test will need to be added to `dmn-js` package. I will do that.

Reply:
Done via https://github.com/bpmn-io/dmn-js/pull/678/commits/af2cb2c08840ec9eb26a6c6075817251e1675dab

Reply:
:+1: